pandas/tests/io/excel/test_readers.py

```diff
@@ -596,7 +596,7 @@ def test_read_from_file_url(self, read_ext, datapath):
             # fails on some systems
             import platform
 
-            pytest.skip("failing on {}".format(" ".join(platform.uname()).strip()))
+            pytest.skip(f"failing on {' '.join(platform.uname()).strip()}")
```

Comment:
I think the complexity of the substitution (measured by number of expressions) could warrant a separate assignment for clarity

Reply:
I separated the expression